docs/reference/ml/apis/evaluateresources.asciidoc

```diff
@@ -0,0 +1,50 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-evaluate-dfanalytics-resources]]
+=== Evaluate {dfanalytics} resources
+
+The evaluation object defines the type of evaluation you want to perform. An 
+evaluation configuration object has the following properties:
+
+`binary_soft_classification`::
+  (object) Configures a binary soft classification type evaluation.
+  
+  
+[float]
+[[binary-sc-resources]]
+==== Binary soft classification resources
+
+A binary soft classification object has the following properties:
+
+`actual_field` (Required)::
+  (string) The field of the `index` which contains the `ground 
+  truth`. The data type of this field can be boolean or integer. If the data 
+  type is integer, the value has to be either `0` (false) or `1` (true).
+
+`predicted_probability_field` (Required)::
+  (string) The field of the `index` that defines the probability of whether the 
+  item is an outlier or not. It is the field that contains the results of the 
```

Comment:
We should not use the term `outlier` here. Instead, we should explain it is the probability of whether the item is of the class in question or not.